[1.22.0] release notes (#317)
replicatedhq/kots.io: content/release-notes/1.22.0.md

---
date: 2020-10-30
linktitle: "1.22.0"
title: "1.22.0"
kubernetes: "1.17, 1.18, and 1.19"
---

{{<features>}}

>   * Added support for disaster recovery backups and restores. Restores can be created both on the same instance or on a different instance. Embedded clusters do not support this feature yet.

Reviewer: ```suggestion
  * Added beta support for disaster recovery backups and restores of instance snapshots, which include the Admin Console and all installed applications. Backups can be restored to the original instance or to a different instance. Instance snapshots do not yet support embedded clusters.
```

{{</features>}}

{{<fixes>}}
* Fixed a bug where base64 encoded application icons do not render in airgapped installations.

> * Fixed a total snapshot capture time.

Reviewer: ```suggestion
* Fixed reporting of total snapshot capture time.
```

> * Fixed remaining seconds when restoring volumes from application backups.

Reviewer: ```suggestion
* Fixed reporting of remaining seconds when restoring volumes from application backups.
```

> * Fixed a bug in the KOTS Admin Console UI where a failure from "Check for updates" was not properly indicated.

Reviewer: Hmm... this doesn't make much sense. What's the failure? Is this about better error reporting?

Author: i believe so yeah
